### ISSUE TITLE
API documentation not rendering

### ISSUE THREAD

**Issue by mivade** [pallets-eco/flask-caching#510]
The API documentation on readthedocs is not rendering:
![image](https://github.com/pallets-eco/flask-caching/assets/2805515/da4a02aa-613f-407b-9783-9e299d37754c)

Additionally the page titles indicate Flask-Caching version 1.0.0.

Environment:

- Python version: N/A
- Flask-Caching version: 2.0.2

**Comment by snarfed** [comment]
https://flask-caching.readthedocs.io/ , right? Looks like it's ok now.

**Comment by mivade** [comment]
No, the API documentation located here does not actually contain API documentation:

https://flask-caching.readthedocs.io/en/latest/api.html

**Comment by kielnino** [comment]
I was also looking for the API documentation...

**Comment by jbbqqf** [comment]
Hi! Triaging older issues — the "version 1.0.0 in titles" part of this is reproducible and there's a small PR up for it now.

Evidence:
- Title still reads "Flask-Caching 1.0.0 documentation" today:
  ```
  $ curl -sL https://flask-caching.readthedocs.io/en/latest/ | grep "<title>"
  <title>Flask-Caching — Flask-Caching 1.0.0 documentation</title>
  ```
  (verified 2026-05-22).
- Root cause: [`docs/conf.py:71-73`](https://github.com/pallets-eco/flask-caching/blob/main/docs/conf.py#L71-L73) hardcodes:
  ```python
  version = "1.0.0"
  release = "1.0.0"
  ```
- "API documentation not rendering" looks resolved separately — the page now renders at `/en/latest/api/` (200 OK), though the title still shows 1.0.0 as above.

I've opened #645 to read these from `importlib.metadata.version("Flask-Caching")` so they always match the released package. If the title fix lands, this issue can be closed (or kept open if you'd like to scope it just to the API-page issue, since that part seems already addressed by the dirhtml output).

Happy to follow guidance from a maintainer either way.

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the references above were verified manually.*

**Comment by mivade** [comment]
Yes, it looks like the API docs actually render now. Closing.
